src/coremedia.cs

```diff
@@ -166,8 +168,17 @@ interface CMSampleAttachmentKey {
 		[iOS (11,0), Mac (10,13), TV (11,0)]
 		[Field ("kCMSampleBufferAttachmentKey_CameraIntrinsicMatrix")]
 		NSString CameraIntrinsicMatrixKey { get; }
+
+		[iOS (13,0), Mac (10,15), TV (13,0), Watch (6,0)]
+		[Field ("kCMSampleAttachmentKey_AudioIndependentSampleDecoderRefreshCount")]
+		NSString AudioIndependentSampleDecoderRefreshCountKey { get; }
+
+		[Mac (10,10), Watch (6,0)]
```

Comment:
this shows up in iOS/tvOS API diff - is it available on those platforms ? if not then `[NoiOS][NoTV]`

Reply:
Present in iOS 8+ and tvOS 9+ so attributes are not needed.